next.config.js

```diff
@@ -13,6 +13,10 @@ const nextConfig = {
     formats: ["image/webp"],
   },
   compress: true,
+  i18n: {
```

Comment:
This gave me an idea. We could add translations using Weblate or any other translation provider to make it better for those that speak other languages. This should be for another issue, though.